lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -24,10 +25,30 @@ def execute(self) -> int:
             self.config,
             self.logger,
         )
-        crt.copy_result_tb(
+        res = crt.copy_result_tb(
             self.context.token, self.context.access, self.context.metadata
         )
-        return 0
+
+        # success
+        if res.status_code == 201:
+            return 0
+
+        try:
+            msg = res.json()["message"]
+        except requests.exceptions.JSONDecodeError:
+            msg = res.text if res.text else res.reason
+
+        # dup
+        if res.status_code == 200:
+            click.echo(msg, err=True)
+            return 0
+
+        if not res.ok:
```

Comment:
If we got `201`, we returned with 0 at line 34; if we got `200` we returns with 0 at line 44. Technically, there are other ways to get `res.ok` that aren't errors; however as we don't expect them, printing the message would probably be a good idea anyway.

In fact, I might suggest that we handle only `201` separately and that line 42 be `if res.ok` (that is, if we're succeeding but not with `CREATED` we want to print the message), and then we can drop this test because we're by definition "not ok" if we get here.